webpack/move_to_foreman/components/DisabledButtonWithTooltip.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'patternfly-react';
+import TooltipButton from 'react-bootstrap-tooltip-button';
+
+const DisabledButtonWithTooltip = (props) => {
+  const {
+    id, disabled, disabledReason, children, placement, trigger,
+  } = props;
+
+  const tooltipId = `${id}-tooltip`;
+
+  const getButton = () => (
+    <Button disabled={disabled}>
+      {children}
+    </Button>
+  );
+  const getTooltipButton = () => (
+    <TooltipButton
+      disabled={disabled}
+      tooltipId={tooltipId}
+      tooltipText={disabledReason}
+      title={children}
+      tooltipPlacement={placement}
+      trigger={trigger}
+    />
+  );
+
+  return (
+    <span className="btn-container">
+      {disabled ? getTooltipButton() : getButton()}
```

Comment:
Reading the code and readme of the tooltip button package, aren't you duplicating its functionality here?
According to https://github.com/pastaclub/react-bootstrap-tooltip-button I think you can use the component directly.

Reply:
Oops, that was left over from a previous implementation.  I realized when looking into this that I could just delete this entire component and use the tooltip button directly so I did.